doc/changelog.rst

```diff
@@ -11,56 +11,125 @@ Jump to :ref:`SmartRedis Changelog <sr_changelog>`
 SmartSim
 ========
 
+0.6.1
+-----
 
-Development branch
-------------------
-
-To be released at some future point in time
+Released on 15 February, 2024
 
 Description
 
-- Updated tests to address Torch warning
-- Updated GitHub actions to latest versions in CI/CD
-- Dropped Cobalt support
-- Override the sphinx-tabs extension background color
-- Updated SmartSim's machine learning backends
-- Added ONNX support for Python 3.10
-- Added support for Python 3.11
-- Added support for SmartSim with Torch on Apple Silicon
-- Updated install instructions for Apple Silicon
+- Fix line-ending related errors on MacOS (ARM64)
+- Update instructions for Apple Silicon
```

Comment:
Feel free to drop this from the short description since we are already calling out ARM64 support as a new feature at line 27